src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -204,10 +206,27 @@ export default {
 					this.initialLoadComplete = true;
 				}
 				this.loading = false;
+
+				// Add analytics event for loans query result
+				this.trackLoans();
 			}
 		});
 	},
 	methods: {
+		trackLoans() {
+			// Check for matching hits, for example when sorting a single page of results
+			const hits = this.loans.map(l => l.id).sort().join();
```

Comment:
I think we need to preserve the delivered order of the loan ids. Does this sort alter that?

Reply:
The order delivered to analytics?

Reply:
Correct. It should represent the same order in which the loans are shown on the page

Reply:
Do we want to send a new event if the order is different by IDs the same? Previously we checked the IDs to ensure there weren't duplicate lists, regardless of order.

Reply:
Got it, thanks. I made the false assumption that we wanted to check result duplicates in general. Updated!